com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/preprocessor/sub/line/rewriter/impl/CobolLineIndicatorProcessorImplTest.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ */
+package com.ca.lsp.core.cobol.preprocessor.sub.line.rewriter.impl;
+
+import static com.ca.lsp.core.cobol.preprocessor.CobolSourceFormat.FIXED;
+import static com.ca.lsp.core.cobol.preprocessor.ProcessingConstants.COMMENT_TAG;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.COMMENT;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.COMPILER_DIRECTIVE;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.CONTINUATION;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.NORMAL;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.DEBUG;
+import static com.ca.lsp.core.cobol.preprocessor.ProcessingConstants.WS;
+import static org.junit.Assert.assertEquals;
+
+import com.ca.lsp.core.cobol.preprocessor.sub.CobolLine;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class CobolLineIndicatorProcessorImplTest {
+  private static final String EMPTY_STRING = "";
+  private CobolLine CONTINUATION_LINE = new CobolLine();
+  private CobolLine TRAILING_QUOTE_CONTINUATION_LINE = new CobolLine();
+  private CobolLine EMPTY_CONTINUATION_LINE_TO_TEST = new CobolLine();
+  private CobolLine SINGLE_CONTINUATION_LINE_TO_TEST = new CobolLine();
+  private CobolLine GOOD_CONTINUATION_LINE_TO_TEST = new CobolLine();
+  private CobolLine BAD_CONTINUATION_LINE_TO_TEST = new CobolLine();
+  private CobolLine COMMENT_LINE_TO_TEST = new CobolLine();
+  private CobolLine CONTINUATION_LINE_WITH_END_QUOTES = new CobolLine();
+  private CobolLine COMPILER_DIRECTIVE_LINE_TO_TEST = new CobolLine();
+  private CobolLine NORMAL_LINE_TO_TEST = new CobolLine();
+  private CobolLine DEBUG_LINE_TO_TEST = new CobolLine();
```

Comment:
These fields may be converted to local variables

Reply:
Done